recipe/meta.yaml

```diff
@@ -5,35 +5,24 @@
 # below since the only purpose of that command is to trigger the installation of the
 # dependencies.
 
-{% set version = "1.0.0" %}
+{% set version = "1.2.4" %}
 
 package:
   name: glueviz
   version: {{version}}
 
-source:
-  fn: glueviz-{{version}}.tar.gz
-  url: https://pypi.io/packages/source/g/glueviz/glueviz-{{version}}.tar.gz
-  sha256: 89db2b2331eb74f221251b129f6ce6b507f14101caea993ee0926ceeeaaf68ab
-
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  # Skipping unnecessary linux platforms that don't need viz.
+  skip: True  # [py<36 or ppc64le or s390x]
 
 requirements:
-  build:
-    - python
-    - pip
-    - setuptools >=30.3
-    - setuptools_scm
+  is_meta_pkg: True
```

Comment:
I don't see this documented in conda build documentation, but it doesn't hurt.

Reply:
This was taken from our Anaconda Distribution.  I think you are correct that it doesn't matter.